packages/preview/unofficial-sdu-thesis/0.2.0/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2011-2025 The Bootstrap Authors
```

Comment:
Sorry, I just noticed, but I think you forgot to update that line after copy/pasting.